corehq/apps/api/resources/v0_5.py

```diff
@@ -1059,11 +1059,14 @@ def prepend_urls(self):
 
 
 class NavigationEventAuditResource(HqBaseResource, ModelResource):
-
+    local_date = fields.DateField(attribute='local_date', readonly=True)
+    UTC_start_time = fields.DateTimeField(attribute='UTC_start_time', readonly=True)
+    UTC_end_time = fields.DateTimeField(attribute='UTC_end_time', readonly=True)
```

Comment:
Shouldn't there be a field for `user`?  Or is that inferred from the model given `Meta.fields`?

Reply:
That was inferred from Meta.fields, but updated per your other comment [15eb2b4](https://github.com/dimagi/commcare-hq/pull/32922/commits/15eb2b48b5986c3a738c614cf3e8949b0aac7dde)